src/cast_core/src/CollectionScanner.cpp

```diff
@@ -513,7 +517,6 @@ CollectionScanner::CollectionScanner(genny::ActorContext& context)
       _index{WorkloadContext::getActorSharedState<CollectionScanner, ActorCounter>().fetch_add(1)},
       _runningActorCounter{
           WorkloadContext::getActorSharedState<CollectionScanner, RunningActorCounter>()},
-      _generateCollectionNames{context["GenerateCollectionNames"].maybe<bool>().value_or(false)},
```

Comment:
There are a number of existing workloads that have `GenerateCollectionNames` in the yaml - do those need to be updated?

Reply:
There is no need for the instance variable here, we were setting everything up in the phase config anyway.

The only behaviour that this changes is to add some error checking if the configuration is incorrect (i.e. GenerateCollectionNames true and an invalid count). Up to now it would have failed with a divide by zero error so the workloads **should** be ok.

I've doubled checked the places where it is used and removed the extra instance  of the configuration (which is a not applied anyway). Plus I am running a patch.